cypress/views/filtrableTable.js

```diff
@@ -0,0 +1,79 @@
+const TOOLBAR = 'div[id="ins-primary-data-toolbar"]';
+const PAGINATION = 'div[data-ouia-component-type="PF4/Pagination"]';
+const PAGINATION_MENU = `${TOOLBAR} ${PAGINATION} div[data-ouia-component-type="PF4/PaginationOptionsMenu"]`;
+const PAGINATION_NEXT = `${TOOLBAR} ${PAGINATION} button[data-action="next"]`;
+const CHIPS = `${TOOLBAR} div[data-ouia-component-type="PF4/ChipGroup"]`;
+const EMPTY_STATE = 'table .pf-c-empty-state';
+const TOGGLE_CHECKBOX = `${TOOLBAR} [data-ouia-component-id="clusters-selector-toggle-checkbox"]`;
+const TOGGLE_CHECKBOX_TEXT = `${TOOLBAR} #toggle-checkbox-text`;
+
+const filtrableTable = {
+  isDisplayed: (id) => {
+    cy.get(`div[id=${id}]`)
+      .within(($div) => {
+        cy.get(TOOLBAR).should('have.length', 1);
+        cy.get('table').should('have.length', 1);
+        cy.get('div[data-ouia-component-type="RHI/TableToolbar"]').should(
+          'have.length',
+          1
+        );
+      })
+      .should('have.length', 1);
+  },
+  rows: () => {
+    return cy
+      .get('table tbody[role=rowgroup]')
+      .find('[data-ouia-component-type="PF4/TableRow"]');
+  },
```

Comment:
Nit-pick: function closures with the single `return` statements or single expression can omit curly brackets. E.g., in this case:
```suggestion
  rows: () => cy
      .get('table tbody[role=rowgroup]')
      .find('[data-ouia-component-type="PF4/TableRow"]'),
```

(from the code below this can be applied for the most of the functions)

Reply:
yes, thanks. Should this also apply to functions without return statement? E.g. 

```
checkRowCounts: (n) => {
    cy.get('table tbody[role=rowgroup]')
      .find('[data-ouia-component-type="PF4/TableRow"]')
      .should('have.length', n);
  }
```

Reply:
@ikerreyes, well, this is actually the case when two approaches **can** lead to different results. When the JS function doesn't have a `return` statement, then this function returns `undefined`. However, once you get rid of curly brackets, the return statement will be the result of the expression (in your case, cy. ... .should(...)). If the result of the function is not consumed anywhere, this is safe to change.

Reply:
Thanks for the explanation. And would you prefer to change?

Reply:
@ikerreyes, in this case it is safe, I would prefer changing it